### PR TITLE
Add Function To Run Commands Synchronously And Return Output

### PR DESCRIPTION
**Pull request by pamaddox**
In mtevbusted.... we need a way to run an external process and return
the output of said process. Added functions to allow for this.

### CODE REVIEW

File: test/mtevbusted/child.lua [hunk 1]
```diff
@@ -57,6 +57,42 @@ function TestProc:waitfor(key, timeout)
...
+  end
+  mtev.notify(self.start, true)
+  local key, data = mtev.waitfor(self.output,self.timeout)
+
+  self.proc:kill()
+  self.proc:wait(10)
+  self.proc = nil
+
+  return data
+end
```
Comment (aidancully) [inline test/mtevbusted/child.lua]: I think this should capture the exit status (from `self.proc:wait()`), the standard output (like you have) and the stderr output. Also, `self.proc:wait(10)` - I think the `10` should be a parameter, maybe in `props`?

Comment (pamaddox) [reply test/mtevbusted/child.lua]: self.proc:wait(10) is a direct copy/paste from the "start()" function - if we change it here, we'd probably want to change it there as well so we don't have inconsistent behavior.

As for capturing stderr - would have to look into that a bit, not sure how that'd work with the current structure to read data/notice errors.

Comment (pamaddox) [reply test/mtevbusted/child.lua]: Should be addressed now

File: test/mtevbusted/child.lua [hunk 2]
```diff
@@ -57,6 +76,41 @@ function TestProc:waitfor(key, timeout)
...
+    self.proc:kill()
+    ret = self.proc:wait(10)
+  end
+  if ret == nil then
+    error("Couldn't quit process")
+  end
+
+  self.proc = nil
+
+  return stdout_data, stderr_data
```
Comment (aidancully) [inline test/mtevbusted/child.lua]: `return ret, stdout_data, stderr_data`

**Comment by aidancully** [comment]
LGTM, but this is more @postwait's area

**Pull request merged**